src/Kmd.Logic.Email.Client/ServicesMessages/EmailAddressDetails.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Kmd.Logic.Email.Client.ServicesMessages
+{
+    public class EmailAddressDetails
```

Comment:
If we can distinguish between `EmailAddress` and this class using some other naming convention that would be good.

Reply:
As of new will keep it `EmailAddressDetails` only will change it later